ext/libzip/zip_source_file_win32_utf16.c

```diff
@@ -41,6 +41,29 @@ static HANDLE __stdcall utf16_create_file(const char *name, DWORD access, DWORD
 static void utf16_make_tempname(char *buf, size_t len, const char *name, zip_uint32_t i);
 static char *utf16_strdup(const char *string);
 
+#ifdef MS_UWP && !defined(__LIBRETRO__)
```

Comment:
Why would we want to exclude LIBRETRO?

Also, you may want to propose these changes to https://github.com/nih-at/libzip.

-[Unknown]